cmd/wallet-web/src/pages/Dashboard.vue

```diff
@@ -104,8 +102,8 @@ export default {
   },
   data() {
     return {
-      cards: [],
-      cjson: credentialDisplayData,
+      credentials: [],
```

Comment:
you don't need credentials as instance variable here, not even for credential detail screen.

pass processed credential to credential detail screen not whole credential (for credential detail PR).

Reply:
removed, make sense